.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Tag and pre-deploy consignment export
```

Comment:
Is this the correct name for this workflow?

Reply:
Yep absolutely. I've changed it anyway though :grin: